src/data/posts.json

```diff
@@ -1,32 +1,47 @@
 [
   {
-    "title": "New board support: the UP board",
-    "tags": ["announcement"],
-    "text": "The UP board is x86-based with a Raspberry Pi compatible 40-pin connector, designed for 'professional makers' who need a reliable, high-performing device to build and grow their projects.",
-    "link": "https://resin.io/blog/up-board-support/",
-    "image": "https://resin.io/blog/content/images/2017/03/UPSlide3Right-EVT-3.png",
+    "title": "LEGO Macintosh Classic with e‑paper display",
```

Comment:
Similarly, let's change the title to "Build a LEGO Macintosh Classic with e-paper display"